semgrep-core/Makefile

```diff
@@ -36,6 +36,9 @@ clean:
 #
 .PHONY: test
 test: all
+	# ensure that '--help' is available and that we don't load test data
```

Comment:
I don't understand this comment. Why do we need to test --help? and what is this load of test data?

Reply:
The test executable has options. There's a `-e` option that we're now using. Learning about it requires running `--help`.

Reply:
I improved the comment.